### PR TITLE
Improved empty section handling

### LINKED ISSUE #11: Top hits don't always emtpy out
See #10.

### PR DESCRIPTION
**Pull request by nchapman**
- Hiding top hits when empty
- Hiding activity when empty
- Showing searchTerm in search suggestions when empty

Fixes #11.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -13,3 +13,4 @@ rules:
   no-use-before-define: [2, nofunc]
   quotes: [2, single]
   strict: 0
+  no-unused-expressions: 0
```
Comment (pdehaan) [inline .eslintrc:16]: Oh. My. Glob.
MY SORTING! RUINED!

Comment (nchapman) [inline .eslintrc:16]: lol so sorry! I'll fix!

**Comment by pdehaan** [comment]
Man I really love this Babel/ES6 codebase. So pretty!

File: .eslintrc (hunk 1 above)
Comment (pdehaan) [inline .eslintrc:16]: I need to build ESLint-Lint which lints the YAML config files for ... things.

**Pull request merged**